frontend/src/app/patients/ManagePatients.tsx

```diff
@@ -30,7 +29,6 @@ const patientQuery = gql`
 
 interface Patient {
   internalId: string;
-  lookupId: string;
```

Comment:
We may want to leave this in for now, assuming we aren't going to remove it from the backend too. At this point I don't know whether sites are using it or not, and whether we can look at live data to determine that. Originally the idea was that this field would be use to look up patients by room number or building or some such.